chromium_src/net/base/lookup_string_in_fixed_set.cc

```diff
@@ -41,12 +41,16 @@ int LookupSuffixInReversedSet(const unsigned char* graph,
     return kDafsaFound;
   }
 
-  // Recognize .crypto as a known TLD for unstoppable domains support. With
-  // this, when users type *.crypto in omnibox, it will be parsed as
-  // OmniboxInputType::URL input type instead of OmniboxInputType::UNKNOWN,
+  // Recognize .crypto and .eth as known TLDs for decentralized DNS support.
+  // With this, when users type *.crypto or *.eth in omnibox, it will be parsed
+  // as OmniboxInputType::URL input type instead of OmniboxInputType::UNKNOWN,
   // The first entry in the autocomplete list will be URL instead of search.
-  if (base::EndsWith(host, unstoppable_domains::kCryptoDomain)) {
-    *suffix_length = strlen(unstoppable_domains::kCryptoDomain) - 1;
+  if (base::EndsWith(host, decentralized_dns::kCryptoDomain)) {
+    *suffix_length = strlen(decentralized_dns::kCryptoDomain) - 1;
```

Comment:
nit: I know it's not from this PR, but at some point it would be good to rename this to kUnstoppableDomain because crypto is overloaded enough already :)

Reply:
Personally I felt kCryptoDomain is more straightforward for “.crypto” than kUnstoppableDomain, so I'll keep it for now.

Reply:
OK np here :)